upload_multiple_files_to_single_record.py

```diff
@@ -0,0 +1,200 @@
+import json
+import requests
+import hashlib
+from pathlib import Path
+from defusedxml.lxml import tostring
+from lxml.builder import E
+import mimetypes
+
+
+SITE_URL = 'https://library.tind.io/'
+API_KEY = ''
+OBJECT_STORE_NAME = 'TOS'
+CALLBACK_EMAIL = 'demo@tind.io'
+
+
+APPEND_METADATA = {"245__a": "Test record",
+                   "269__a": "2021-10-14",
+                   "980__a": "DIGITIZED"}
+
+
+def request_presigned_object():
+    """Returns a presigned object"""
+    url = SITE_URL + 'storage/presigned_post?location=' + OBJECT_STORE_NAME
+    headers = {'Authorization': 'Token ' + API_KEY}
+    r = requests.post(url, headers=headers)
+    response_object = json.loads(r.content)
+    return response_object
+
+
+def upload_single_file(file_name, presigned_response):
+    """Post a file to S3 and returns the upload response"""
+    url = presigned_response["data"]["url"]
+    headers = {'x-amz-acl': 'private'}
+    data = presigned_response["data"]["fields"]
+    files = {'file': open(file_name, 'rb')}
+    # Time out if the file is not uploaded in 600 seconds
+    upload_response = requests.post(url, headers=headers, files=files, data=data, timeout=600)
+
+    print(upload_response.status_code)
+    return upload_response
+
+
+def verify_checsum(file_path, upload_response):
+    """
+    Comparing the checksum of the local file with the returned checksum/Etag
+    stored in the upload response
+    """
+    local_checksum = hashlib.md5(open(file_path, 'rb').read()).hexdigest()
+
+    etag = upload_response.headers.get('Etag', '')
+    upload_checksum = etag.replace('"', '')
+
+    if local_checksum != upload_checksum:
+        print('Checksum is not the same! %s' % (file_path,))
+        return False
+    return local_checksum
+
+
+def create_subfield(sub_code):
```

Comment:
I feel like the subfield content should be handled here, not in the `create_datafield` function

```suggestion
def create_subfield(sub_code, content):
```